includes/validation/class-amp-validated-url-post-type.php

```diff
@@ -1936,11 +1955,31 @@ public static function print_status_meta_box( $post ) {
 							echo '</p></div>';
 						}
 						?>
-						<?php self::display_invalid_url_validation_error_counts_summary( $post ); ?>
+
+						<?php
+						$counts = self::count_invalid_url_validation_errors( self::get_invalid_url_validation_errors( $post ) );
+
+						if ( 0 < ( $counts['new_rejected'] + $counts['new_accepted'] ) ) {
+							?>
+							<strong id="amp-invalid-markup" class="status-text">
+								<span class="amp-icon amp-warning"></span>
+								<?php esc_html_e( 'Invalid markup not reviewed', 'amp' ); ?>
+							</strong>
+							<?php
+							esc_html_e(
+								'Review each instance of invalid markup and determine whether the action of removing
+								(or keeping) is approved. You can change the status and click preview to see what
+								impact the invalid markup has on the page. Invalid markup that is not approved is
+								markup which you may not have seen yet or which you want to fix directly in
+								underlying code.',
```

Comment:
I think the linebreaks perhaps should be removed? I don't recall translation strings having these. The extra tabs may confuse translators. I think it should all be on one line.